docs/notebooks/figanos_docs.ipynb

```diff
@@ -54,12 +54,32 @@
   },
```

Comment:
<p>need to be clearer that there is no need to pass a logo argument if you have installed the ouranos logo.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/figanos/pull/119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>